sdk/js-query/src/query/solana.test.ts

```diff
@@ -204,7 +204,9 @@ describe("solana", () => {
     expect(sar.results[0].lamports.toString()).toEqual(
       BigInt(1461600).toString()
     );
-    expect(sar.results[0].rentEpoch.toString()).toEqual(BigInt(0).toString());
+    expect(sar.results[0].rentEpoch.toString()).toEqual(
+      BigInt("18446744073709551615").toString()
```

Comment:
🤔

Reply:
*sigh*... the older version of `solana-test-validator` just returned a rentEpoch of `0` and the new one returns `18446744073709551615`. It is nice to have a value now.

Reply:
Is it worth defining a constant with a meaningful name rather than using the magic value everywhere? Make it easier if they change the value.